pacakges/server/src/core.rs

```diff
@@ -124,37 +126,108 @@ pub struct Requirement{
     pub message: Option<String>,
 }
 
+impl Requirement {
+    fn create_requirement(bank_name: String, credit_requirment: f32) -> Requirement {
```

Comment:
rename to just "create", because in the call we always have the "Requirement::" namespace precceding so adding another '_requirement' suffix to the function is redundant